package.json

```diff
@@ -13,7 +13,7 @@
     "registry": "https://registry.npmjs.org/"
   },
   "license": "MIT",
-  "main": "dist/cjs/index.js",
+  "main": "dist/esm/index.js",
   "module": "dist/esm/index.js",
```

Comment:
@dword-design @wesdevpro I found a good explanation in the documentation of Nuxt:
- <https://nuxt.com/docs/guide/concepts/esm#what-is-native-esm>
- https://nuxt.com/docs/guide/concepts/esm#what-are-valid-imports-in-a-nodejs-context

What we should do will be to change the extension in `module` field from `.js` to `.mjs`.
```json
"module": "dist/esm/index.mjs",
```
To do so, we have to change the Rollup script. But you may try it by changing your copy of `@ntohq/buefy-next` in `node_modules`.